elyra/kfp/bootstrapper.py

```diff
@@ -482,10 +483,19 @@ def execute(self) -> None:
             run_args = ["python3", python_script]
             if self.parameter_pass_method == "env":
                 self.set_parameters_in_env()
-
+            
+            logger.info("----------------------Python logs start----------------------")
             with open(python_script_output, "w") as log_file:
-                subprocess.run(run_args, stdout=log_file, stderr=subprocess.STDOUT, check=True)
-
+                process = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+                for line in iter(process.stdout.readline, b''):
+                    sys.stdout.write(line.decode())
+                    log_file.write(line.decode())
```

Comment:
It is probably best to drop the log_file getting stored in s3.
lets just have them directly into container log .